QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,248 @@
+
+You first need the Bottlerocket root role, which is used by `tuftool` to verify the OVA.
+The following will download and verify the root role itself:
+
+```
+curl -O "https://cache.bottlerocket.aws/root.json"
+sha512sum -c <<<"90393204232a1ad6b0a45528b1f7df1a3e37493b1e05b1c149f081849a292c8dafb4ea5f7ee17bcc664e35f66e37e4cfa4aae9de7a2a28aa31ae6ac3d9bea4d5  root.json"
+```
+
+Next, set your desired version and variant, and download the OVA:
+```
+VERSION="v1.1.0"
+VARIANT="vmware-k8s-1.20"
+
+tuftool download . --target-name bottlerocket-${VARIANT}-x86_64-${VERSION}.ova \
+   --root ./root.json \
+   --metadata-url "https://updates.bottlerocket.aws/2020-07-07/${VARIANT}/x86_64/" \
+   --targets-url "https://updates.bottlerocket.aws/targets/"
+```
+
+## Upload the OVA
+
+Once you have downloaded the OVA, you can upload it to vSphere.
+
+The first command generates a spec file (`bottlerocket_spec.json` in this case) using the OVA and gives you few options for your deployment.
+```
+govc import.spec <Bottlerocket OVA > bottlerocket_spec.json
+```
+
+The spec will look similar to this:
+```
+{
+  "DiskProvisioning": "flat",
+  "IPAllocationPolicy": "dhcpPolicy",
+  "IPProtocol": "IPv4",
+  "NetworkMapping": [
+    {
+      "Name": "VM Network",
+      "Network": ""
+    }
+  ],
+  "MarkAsTemplate": false,
+  "PowerOn": false,
+  "InjectOvfEnv": false,
+  "WaitForIP": false,
+  "Name": null
+}
+```
+
+You must enter valid values for the `Network` and `Name`.
+`Network` must correspond to the network in which you intend to deploy this VM.
+`Name` can be anything you want, it is the name of the VM that will be created.
+
+Once you've edited the file, you can use it to upload the OVA!
+```
+govc import.ova -options=bottlerocket_spec.json <bottlerocket...ova>
+```
+
+> Note: `govc import.ova` also takes a `-name=` argument that allows you to use the same spec file but override the VM name on future uploads.
+> For example: `govc import.ova -options=bottlerocket_spec.json -name="BottlerocketIsNeat" <bottlerocket...ova>`
+
+### Creating a template
+
+If you intend to run many identical VMs, you can mark the OVA you just uploaded as a template.
+You **must** to do this before you boot and configure Bottlerocket, since user data configuration will be unique between nodes!
+You can think of a template as a "golden" image, allowing you to create many VMs without affecting the "golden" image.
+
+The following command marks the VM as a template:
+```
+govc vm.markastemplate <YOUR VM NAME>
+```
+
+## Gathering cluster info
+
+This section will help you gather cluster information needed to configure Bottlerocket via user data.
+
+#### API Server
+This is the address (including port) of the control plane.
+You can run the following to get it:
+```
+ kubectl --kubeconfig=<your kubeconfig> cluster-info
+```
+The value you need will be found after `Kubernetes control plane is running at:` and should start with `https://` and end with a port.
```

Comment:
Ideally we'd give them a `kubectl | awk` command that they could paste in, something like:

```
export API_SERVER="$(kubectl ... | awk ...)
```